src/main/java/com/booster/server/NoteDto.java

```diff
@@ -0,0 +1,12 @@
+package com.booster.server;
+
+import lombok.Getter;
+import lombok.Setter;
+import lombok.experimental.Accessors;
+
+@Getter
+@Setter
+@Accessors(chain = true)
+public class NoteDto {
+    String content;
```

Comment:
Please add a `private` modifier and a `@ToString`.
The same is required for the `VocabularyEntryDto`.

Reply:
Done